l10n_it_account_balance_eu/README.rst

```diff
@@ -0,0 +1,156 @@
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fl10n--italy-lightgray.png?logo=github
+    :target: https://github.com/OCA/l10n-italy/tree/14.0/l10n_it_account_balance_eu
+    :alt: OCA/l10n-italy
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/l10n-italy-14-0/l10n-italy-14-0-l10n_it_account_balance_eu
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/122/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+**Italiano**
+
+In Odoo è presente il piano dei conti tradizionale e le varie registrazioni
+contabili
+con gli importi ma non è presente una struttura di voci
+che descrivano il Bilancio UE a cui associare le voci del piano dei conti
+tradizionale.
+
+Questo modulo consente di generare un bilancio riclassificato secondo lo schema
+del bilancio UE
+con tutte le voci delle più recenti tassonomie mettendo a disposizione le
+seguenti funzionalità:
+
+* associare ai conti del piano dei conti le voci del Bilancio UE (con preassociazione automatica in fase di installazione del modulo)
+* elaborare le registrazioni contabili di un dato periodo (per esempio l’anno precedente) al fine di ottenere il bilancio contabile in formato UE, facendo gli opportuni controlli di congruenza,  associazione di tutti i conti movimentati nel periodo e quadratura del bilancio. Nei report viene riportato anche il dettaglio dei conti associati ad una certa voce del bilancio UE
+* generare il report di anteprima del Bilancio UE
+* generare la stampa PDF del Bilancio UE
+* generare i file XLSX e XBRL del bilancio (utili per successive rielaborazioni e per trasferimento del bilancio UE al Commercialista)
+
+**English**
+
+In Odoo there is the traditional chart of accounts and the various registrations
+accountants
+with amounts but there is no line item structure
+that describe the EU balance to which to associate the items of the traditional chart of accounts.
+This module allows to generate a reclassified balance sheet according to the format
+of the EU balance
+with all the entries of the most recent taxonomies by making available the
+following features:
+
+* associate the items in the Chart of Accounts with the items in the EU Budget (with automatic pre-association during module installation)
+* process the accounting records of a given period (for example
+  last year) in order to obtain the accounting balance sheet in EU format, carrying out the appropriate congruence checks, association of all the accounts moved in the period and balancing of the balance sheet. The reports also show the details of the accounts associated with a certain item of the EU balance
+* PDF printout of the EU Balance
```

Comment:
Non ho confrontato il significato in italiano/inglese, ho solo notato che nell'inglese manca un punto

Reply:
Mancava il punto della visualizzazione HTML nell'inglese. Con l'occasione ho corretto dal readme in inglese tutte le occorrenze di "balance"